### ISSUE TITLE
regulate HPO configs according to input

### ISSUE THREAD

**Issue by borauyar** [BIMSBbioinfo/flexynesis#71]
The default configuration may not be suitable for all dataset sizes (in terms of sample sizes or feature sizes). The input dim range could be determined as a function of number of available features; the batch size range could be set according to the number of samples in the training data. Otherwise, training may not work for datasets that are smaller than the batch size range.
